lib/aws-record/record/attributes.rb

```diff
@@ -56,6 +56,19 @@ def to_h
         @data.hash_copy
       end
 
+      # Returns a string representation of the key attributes
+      #
+      # @return [String] String of key attributes that can be used to query a model
+      def to_param
+        hkey = public_send(self.class.hash_key)
+        if self.class.range_key
+          rkey = public_send(self.class.range_key)
+          "#{hkey}:#{rkey}"
```

Comment:
Can we document why this is the separator? It's very possible and valid for these key values to contain the ":" character. Depending on what the API contract is, we may need to do more here.